src/components/Datatable/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+
+import Pagination from './Pagination';
+import { PaginationProps } from './Pagination.types';
+
+export default {
+  title: 'components/Datatable/components/Pagination',
+  component: Pagination,
+} as Meta;
+
+export const Default: Story<PaginationProps> = ({ ...args }) => {
```

Comment:
Needs to be copy of `args`? Isn't just passing args down enough? `Story = (args) => {}`